lib/util/convertSatoshiToCredits.js

```diff
@@ -0,0 +1,7 @@
+const RATIO = 1000;
+
```

Comment:
It looks more like business logic rather than util. Let’s put it to `lib/identity`?